# HG changeset patch
# User Rik <rik@octave.org>
# Date 1349982808 25200
#      Thu Oct 11 12:13:28 2012 -0700
# Node ID bd947371c2c97d12324c3c8cd6488e072fc737b2
# Parent  dca1f2bcccf30ab79966b65370e9bd46b1681138
Export doc-cache in distribution tarball (bug #37542)

* doc/interpreter/Makefile.am: Add doc-cache to EXTRA_DIST target.

* doc/interpreter/mk_doc_cache.m: Define save_header_format_string so
that user and host information isn't exported into tarball.

diff --git a/doc/interpreter/Makefile.am b/doc/interpreter/Makefile.am
--- a/doc/interpreter/Makefile.am
+++ b/doc/interpreter/Makefile.am
@@ -244,16 +244,17 @@ spellcheck: $(SPELLCHECK_FILES)
 	else \
 		echo "Spellcheck passed"; \
 	fi
 .PHONY: spellcheck
 
 EXTRA_DIST = \
   config-images.sh \
   contributors.in \
+	doc-cache \
   find-docstring-files.sh \
   images \
   images.awk \
   images.mk \
   macros.texi \
   mk_doc_cache.m \
   mkcontrib.awk \
   munge-texi.pl \
diff --git a/doc/interpreter/mk_doc_cache.m b/doc/interpreter/mk_doc_cache.m
--- a/doc/interpreter/mk_doc_cache.m
+++ b/doc/interpreter/mk_doc_cache.m
@@ -116,9 +116,10 @@ for i = 2:n
   cache{1,k} = symbol;
   cache{2,k} = doc;
   cache{3,k} = first_sentence;
   k++;
 endfor
 
 cache(:,k:end) = [];    # delete unused pre-allocated entries
 
+save_header_format_string (["# doc-cache created by Octave " OCTAVE_VERSION ", %a %b %d %H:%M:%S %Y %Z"]);
 save ("-text", output_file, "cache");
